### DISCUSSION TITLE
Hybrid Model

### DISCUSSION THREAD

**Discussion by MScharmach** [Virtual-FCS/VirtualFCS#46] [General]
Dear VFCS Team,

I was wondering if the hybrid power train model of the tool works or if they are any attempts ongoing atm?

Best regards,
Marcel

**Comment by benjamsy** [comment]
Hi Marcel,

Can you clarify what you mean with "hybrid power train model"? Do you have issues with one of the example models? If so: what version of OM and Virtual-FCS are you on?

**Reply by MScharmach** [reply]
Hi Ben,

thank you for the reply. I am using OM 1.20 and Version-FCS Version: 2.0.0, 2023-01-19; 
Trying to simulate "ParallelHybridPowerTrain" implemented into a model supplied by vehicle data, which didn´t work. Before going into detailed debugging I was wondering if the submodel  "ParallelHybridPowerTrain" was fully set up. Especially because I saw some open issues on this topic (https://github.com/Virtual-FCS/VirtualFCS/issues/22).

**Reply by benjamsy** [reply]
Hi,

I was able to simulate with minor changes following your description. I would need some more information about what goes wrong to be able to help you. 

There is also a lot of changes in an upcoming update, which will be released soon. This might also solve your problems. 

Best,
